### PR TITLE
code refactor to make bugs less likely (HPC-6137)

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
jenkins test this please

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 # Packages
 *.eggs
+.eggs
```
Comment (wdpypere) [inline .gitignore]: *.eggs should catch this no?

File: lib/vsc/jobs/moab/checkjob.py [hunk 2]
```diff
@@ -130,6 +130,6 @@ def default(self, obj):
 
 class SshCheckjob(Checkjob, SshMoabCommand):
     """
-    Allows for retrieving showq information through an ssh command to the remote master
+    Allows for retrieving checkjob information through an ssh command over a remote master
```
Comment (wdpypere) [inline lib/vsc/jobs/moab/checkjob.py]: over a ? correct Flenglish?

**Comment by wdpypere** [comment]
@JensTimmerman needs tests.

**Pull request merged**